src/screens/About/About.scss

```
@@ -0,0 +1,35 @@
+@import "../../colors";
+
+.about {
+  width: 100%;
+  height: 100%;
+  grid-area: main;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+
+  .wrapper {
+    width: 90%;
+    display: flex;
+    flex-direction: column;
+    margin-top: 40px;
+
```

We need margin bottom here as well
```suggestion
    margin: 40px 0;
```